internal/pkg/service/buffer/storage/statistics/statistics.go

```diff
@@ -148,39 +129,26 @@ type Value struct {
 	FileGZipSize datasize.ByteSize `json:"fileGZipSize,omitempty"`
 }
 
-type PerAPINode struct {
-	SliceKey key.SliceKey
+type PerSlice struct {
+	SliceKey storage.SliceKey
 	Value    Value
 }
 
 // Aggregated contains aggregated statistics for an object, such as file or export.
 type Aggregated struct {
-	// Buffered field contains summarized statistics for slices in slicestate.Writing, slicestate.Closing, slicestate.Uploading, slicestate.Failed.
-	Buffered Value
-	// Uploaded field contains summarized statistics for slices in slicestate.Uploaded.
-	Uploaded Value
-	// Imported  field contains summarized statistics for slices in slicestate.Imported.
-	Imported Value
-	// Total field contains summarized statistics for slices in all states, Buffered + Uploaded + Imported.
+	// Local field contains summarized statistics for slices in storage.SliceWriting, storage.SliceClosing, storage.SliceUploading.
+	// Statistics match the data on local disks.
+	Local Value
+	// Staging field contains summarized statistics for slices in storage.SliceUploaded.
+	// Statistics match the data in the staging storage.
+	Staging Value
+	// Target  field contains summarized statistics for slices in storage.SliceImported.
+	// Statistics match the data in the target table.
+	Target Value
+	// Total field contains summarized statistics for slices in all states, Local + Staging + Target.
 	Total Value
```

Comment:
Nahradime povodne `category` za `level`, aj co sa tyka nazvov fields v `statistics.Aggregated`.
Nech to je rovnako a jednoznacne.